keepercommander/importer/importer.py

```diff
@@ -138,17 +141,56 @@ def get_folder_path(self):
         return path
 
 
+class RecordField:
+    def __init__(self):
+        self.type = None    # type: Optional[str]
+        self.label = None   # type: Optional[str]
+        self.value = None   # type: any
+
+    def name_key(self):
+        if self.type and self.label:
+            return f'${self.type.lower()}.{self.label.lower()}'
+        elif self.type:
+            return f'${self.type.lower()}'
+        else:
+            return (self.label or '').lower()
+
+    def hash_key(self):
+        value = self.value
+        if value:
+            if type(value) is str:
```

Comment:
No big deal, but this could be written as if isinstance(value, str) or if isinstance(value, (str, ))
pylint prefers isinstance, but I've turned that off in our pylintrc.
It might be good to be consistent.

Reply:
I agree. I checked the difference between isinstance and type. It is stated that is isinstance is preferable.
I started using isinstance.